src/test/java/com/powsybl/openloadflow/sa/WoodburyDcSecurityAnalysisWithActionsTest.java

```diff
@@ -0,0 +1,454 @@
+    }
+
+    @Test
+    void testSaDcPhaseTapChangerTapPositionAction() {
+        Network network = MetrixTutorialSixBusesFactory.create();
+        List<StateMonitor> monitors = createAllBranchesMonitors(network);
+        List<Contingency> contingencies = List.of(new Contingency("S_SO_1", new BranchContingency("S_SO_1")));
+        List<Action> actions = List.of(new PhaseTapChangerTapPositionAction("pstAbsChange", "NE_NO_1", false, 1),
+                new PhaseTapChangerTapPositionAction("pstRelChange", "NE_NO_1", true, -1));
+        List<OperatorStrategy> operatorStrategies = List.of(new OperatorStrategy("strategyTapAbsChange", ContingencyContext.specificContingency("S_SO_1"), new TrueCondition(), List.of("pstAbsChange")),
+                new OperatorStrategy("strategyTapRelChange", ContingencyContext.specificContingency("S_SO_1"), new TrueCondition(), List.of("pstRelChange")));
+
+        SecurityAnalysisResult result = runSecurityAnalysis(network, contingencies, monitors, securityAnalysisParameters,
+                operatorStrategies, actions, ReportNode.NO_OP);
+
+        assertNotNull(result);
+
+        OperatorStrategyResult resultAbs = getOperatorStrategyResult(result, "strategyTapAbsChange");
+        BranchResult brAbs = resultAbs.getNetworkResult().getBranchResult("S_SO_2");
+        OperatorStrategyResult resultRel = getOperatorStrategyResult(result, "strategyTapRelChange");
+        BranchResult brRel = resultRel.getNetworkResult().getBranchResult("S_SO_2");
+
+        // Apply contingency by hand
+        network.getLine("S_SO_1").getTerminal1().disconnect();
+        network.getLine("S_SO_1").getTerminal2().disconnect();
+        // Apply remedial action
+        int originalTapPosition = network.getTwoWindingsTransformer("NE_NO_1").getPhaseTapChanger().getTapPosition();
+        network.getTwoWindingsTransformer("NE_NO_1").getPhaseTapChanger().setTapPosition(1);
+        loadFlowRunner.run(network, parameters);
+        // Compare results
+        assertEquals(network.getLine("S_SO_2").getTerminal1().getP(), brAbs.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getBranch("S_SO_2").getTerminal2().getP(), brAbs.getP2(), LoadFlowAssert.DELTA_POWER);
+
+        // Check the second operator strategy: relative change
+        network.getTwoWindingsTransformer("NE_NO_1").getPhaseTapChanger().setTapPosition(originalTapPosition - 1);
+        loadFlowRunner.run(network, parameters);
+        // Compare results
+        assertEquals(network.getLine("S_SO_2").getTerminal1().getP(), brRel.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getBranch("S_SO_2").getTerminal2().getP(), brRel.getP2(), LoadFlowAssert.DELTA_POWER);
+    }
+
+    @Test
+    void testFastSaDcPhaseTapChangerTapPositionChangeAdmittanceOnlyAlphaNull() {
+        Network network = PhaseControlFactory.createWithOneT2wtTwoLines();
```

Comment:
This test is new but it would be good to run it in fast and normal mode also as a pametrized test.

Reply:
@p-arvy I reopen that comment.
I think that every test on DC security should run on fast DC and default DC unless there is a good reason.
We want to know if there is a difference - and it might be a bug.
As of today, 5 tests filas in normal mode.
It is OK for two of them:
   testFastDcSaWithActionNotOnPst because the limitation is only for fast DC
   testSaDcPhaseTapChangerTapPositionChange because there is an issue in normal DC when connectivity is broken, and this is independant from this PR.  (I have added a todo for me to look at this)

I do not think it should be OK for 
   testFastSaDcPhaseTapChangerTapPositionChangeAdmittanceOnlyAlphaNonNull
   testFastSaDcN2ContingencyOneTapPositionChange
   testSaDcPhaseTapChangerTapPositionChange
I have not looked at the details yet but I suspect it is linked to the fact that tap changes are taken into account in AbstractClosedBranchDcFlowEquationTerm but in a way does doesn work well in all cases. If is is true, that should be fixed before the PR is merged.

Reply:
Current state fied by https://github.com/powsybl/powsybl-open-loadflow/pull/1095/commits/c03d5665c12b1526ebcd99b35cfde5acacb2a123
I keep this thread open as a reminder that all new tests should run in fast dc ad default mode unless there is a justified exception.